build-system/server/lazy-build.js

```diff
@@ -88,6 +89,7 @@ exports.lazyBuildExtensions = async function(req, res, next) {
 exports.lazyBuildJs = async function(req, res, next) {
   const matcher = /\/.*\/([^\/]*\.js)/;
   await lazyBuild(req.url, matcher, jsBundles, doBuildJs, next);
+  await formatExtractedMessages();
```

Comment:
How long does this step take? Any chance you can add step timing inside the function call, similar to how it's done for other build steps?

See `startTime` and `endBuildStep()`

https://github.com/ampproject/amphtml/blob/db49f436b562d99df0d005bf12f8f61c11c4af80/build-system/tasks/compile-jison.js#L44-L59

Reply:
`13 ms` on my machine with a full table.

Reply:
(Added to underlying `formatExtractedMessages`, lmk if you'd like me to hoist it to callers.)

Reply:
Very nice!